airbyte-commons-worker/build.gradle

```diff
@@ -0,0 +1,65 @@
+import org.jsonschema2pojo.SourceType
+
+plugins {
+    id "java-library"
+    id 'com.github.eirnym.js2p' version '1.0'
+}
+
+group 'io.airbyte...'
+version '0.40.14'
```

Comment:
I think that `group` and `version` can be removed from this file.  This is handled by the `allprojects` block in the root `build.gradle` file.